src/interface/window.c

```diff
@@ -402,7 +402,7 @@ void window_set_window_limit(int value)
 rct_window *window_create(int x, int y, int width, int height, rct_window_event_list *event_handlers, rct_windowclass cls, uint16 flags)
 {
 	// Check if there are any window slots left
-	if (RCT2_NEW_WINDOW >= &(g_window_list[gConfigGeneral.window_limit])) {
+	if ((RCT2_NEW_WINDOW - 3) > &(g_window_list[gConfigGeneral.window_limit])) {
```

Comment:
For example, if the window limit is 8, with `>=` only you can have 7 windows at the same time, but with `>`, you can have 8 windows.

Is curious xP, probably a mistake.